pkg/cortex/cortex.go

```diff
@@ -109,6 +110,7 @@ type Config struct {
 	TenantFederation tenantfederation.Config         `yaml:"tenant_federation"`
 
 	Ruler          ruler.Config                               `yaml:"ruler"`
+	RuleStore      rulestore.Config                           `yaml:"rule_store" doc:"hidden"`
```

Comment:
What do you think about naming the YAML (and CLI flags) `ruler_storage` to keep it consistent with `blocks_storage`?

Reply:
👍